fendermint/testing/contracts/Greeter.bin

```diff
@@ -1 +1 @@
6040516100b09190610358565b60405180910390a150565b60605f80546100c9906103a5565b80601f01602080910402602001604051908101604052809291908181526020018280546100f5906103a5565b80156101405780601f1061011757610100808354040283529160200191610140565b820191905f5260205f20905b81548152906001019060200180831161012357829003601f168201915b5050505050905090565b5f604051905090565b5f80fd5b5f80fd5b5f80fd5b5f80fd5b5f601f19601f8301169050919050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52604160045260245ffd5b6101a982610163565b810181811067ffffffffffffffff821117156101c8576101c7610173565b5b80604052505050565b5f6101da61014a565b90506101e682826101a0565b919050565b5f67ffffffffffffffff82111561020557610204610173565b5b61020e82610163565b9050602081019050919050565b828183375f83830152505050565b5f61023b610236846101eb565b6101d1565b9050828152602081018484840111156102575761025661015f565b5b61026284828561021b565b509392505050565b5f82601f83011261027e5761027d61015b565b5b813561028e848260208601610229565b91505092915050565b5f602082840312156102ac576102ab610153565b5b5f82013567ffffffffffffffff8111156102c9576102c8610157565b5b6102d58482850161026a565b91505092915050565b5f81519050919050565b5f82825260208201905092915050565b5f5b838110156103155780820151818401526020810190506102fa565b5f8484015250505050565b5f61032a826102de565b61033481856102e8565b93506103448185602086016102f8565b61034d81610163565b840191505092915050565b5f6020820190508181035f8301526103708184610320565b905092915050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52602260045260245ffd5b5f60028204905060018216806103bc57607f821691505b6020821081036103cf576103ce610378565b5b50919050565b5f819050815f5260205f209050919050565b5f6020601f8301049050919050565b5f82821b905092915050565b5f600883026104317fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff826103f6565b61043b86836103f6565b95508019841693508086168417925050509392505050565b5f819050919050565b5f819050919050565b5f61047f61047a61047584610453565b61045c565b610453565b9050919050565b5f819050919050565b61049883610465565b6104ac6104a482610486565b848454610402565b825550505050565b5f90565b6104c06104b4565b6104cb81848461048f565b505050565b5b818110156104ee576104e35f826104b8565b6001810190506104d1565b5050565b601f82111561053357610504816103d5565b61050d846103e7565b8101602085101561051c578190505b610530610528856103e7565b8301826104d0565b50505b505050565b5f82821c905092915050565b5f6105535f1984600802610538565b1980831691505092915050565b5f61056b8383610544565b9150826002028217905092915050565b610584826102de565b67ffffffffffffffff81111561059d5761059c610173565b5b6105a782546103a5565b6105b28282856104f2565b5f60209050601f8311600181146105e3575f84156105d1578287015190505b6105db8582610560565b865550610642565b601f1984166105f1866103d5565b5f5b82811015610618578489015182556001820191506020850194506020810190506105f3565b868310156106355784890151610631601f891682610544565b8355505b6001600288020188555050505b50505050505056fea264697066735822122016f2baf93879d1f50483aa9b26b1323ba354fd1a58899568601512441cffe1c364736f6c63430008170033
```

Comment:
Should these `.bin` files change? CI passes so maybe it's just a compiler version thing, but just making sure it's not your test version being committed without the source changes which you removed from the PR?

Reply:
I have solc version 0.8.23 which seems to result in a slightly different artifacts, but yeah its compiled from the original contract so it should be good